server.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express')
+const app = express()
```

Comment:
Consider adding error handling middleware to your express server. This can help with debugging and provide more informative error responses. [important]